mesonbuild/interpreter.py

```diff
@@ -104,12 +106,19 @@ def run_command(self, cmd, args, source_dir, build_dir, subdir, mesonintrospect,
             cwd = os.path.join(source_dir, subdir)
         child_env = os.environ.copy()
         child_env.update(env)
+        stdout = subprocess.PIPE
```

Comment:
`stdout = subprocess.PIPE if self.capture else subprocess.DEVNULL`? Splitting initialization in several statements has a risk that one of them will be moved later.

Reply:
Sure.